connect/runtime/src/test/java/org/apache/kafka/connect/storage/FileOffsetBackingStoreTest.java

```diff
@@ -109,6 +121,69 @@ public void testThreadName() {
                 .newThread(EMPTY_RUNNABLE).getName().startsWith(FileOffsetBackingStore.class.getSimpleName()));
     }
 
+    @Test
+    public void testConnectorPartitions() throws Exception {
+        @SuppressWarnings("unchecked")
+        Callback<Void> setCallback = mock(Callback.class);
+
+        JsonConverter jsonConverter = new JsonConverter();
```

Comment:
We can add an accompanying comment here:

```suggestion
        // This test actually requires the offset store to track deserialized source partitions, so we can't use the mock converter we created in setup()
        JsonConverter jsonConverter = new JsonConverter();
```